src/components/shared/loadingSpinner/loadingSpinner.css

```diff
@@ -0,0 +1,16 @@
+
+.loader {
+  border: 16px solid #f3f3f3; /* Light grey */
+  border-top: 16px solid #f2a22c; /* Blue */
+  border-radius: 50%;
+  width: 120px;
+  height: 120px;
+  animation: spin 2s linear infinite;
+  position: absolute;
+  margin-left: 45%;
```

Comment:
you can do:
margin: auto

remove: 
position: absolute, (this one is tricky beacuse you will always have to remember to set parent position to relative)
margin-left